.travis.yml

```diff
@@ -9,12 +9,14 @@ dist: trusty
 language: c
 before_install:
  - sudo apt-get update
- - sudo apt-get install -y python python3-distutils-extra python3-setuptools python-dev python3-dev python-coverage pep8 pyflakes python-apt python3-apt python-mock python3-mock lsb-release dh-systemd
+ - sudo apt-get build-dep unattended-upgrades
+ - sudo apt-get install -y dh-systemd fakeroot python-mock python3-mock
  - which python3
  - sudo apt-get install python3-pip
  - sudo python3 -m pip install -U mypy
 # travis is "funny" it has a non-distro python3 that will not find
 # python modules installed via apt-get
 script:
-  - (cd test ; PATH=/usr/bin:/usr/local/bin:$PATH make)
-  - sudo unattended-upgrades -v
+  - dpkg-buildpackage -d -us -uc -rfakeroot
```

Comment:
Looks great! Maybe a small comment like `# note that tests are run during package build`  ?

Reply:
done